### ISSUE TITLE
fence_pve fails with stacktrace

### ISSUE THREAD

**Issue by howdoicomputer** [ClusterLabs/fence-agents#38]
Ubuntu 14.04

```
root@dev-nfs-archive-1001:/usr/sbin# fence_pve --ip=<ADDRESS> --username=root@pam --password=<PASSWORD> --action=status --plug=187

Traceback (most recent call last):
  File "/usr/sbin/fence_pve", line 184, in <module>
    main()
  File "/usr/sbin/fence_pve", line 175, in main
    options["auth"] = get_ticket(options)
  File "/usr/sbin/fence_pve", line 82, in get_ticket
    result = send_cmd(options, "access/ticket", post=post)
  File "/usr/sbin/fence_pve", line 110, in send_cmd
    if opt.has_key("--ssl") or opt.has_key("--ssl-secure"):
NameError: global name 'opt' is not defined
```

**Comment by howdoicomputer** [comment]
I just realized that the fix for this bug went in with `4.0.21`

**Comment by marxsk** [comment]
ok, so I'm closing this issue
